### ISSUE TITLE
Project Death ?

### ISSUE THREAD

**Issue by bastienlemaitre** [gallayl/FuryTech.OdataTypescriptServiceGenerator#9]

**Comment by gallayl** [comment]
not really, just out of my scope at the moment, but I plan to continue it

**Comment by NetTecture** [comment]
Which, in March 2018 translates into project death.

Crap. Seems like every decent odata side project in typescript is dying- researchign like nuts for a decent client ;(

**Comment by AnderssonPeter** [comment]
I wish this project had support for react
